branchinfo.txt

```diff
@@ -2,10 +2,10 @@
 # Any line that is not blank and does not start with '#' is interpreted as a variable to set.
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=2
-MINOR_VERSION=0
+MINOR_VERSION=1
```

Comment:
i'm not aware of anything still using branchinfo.txt.  Part of the intention of the buildtools effort was to make everything use dir.props.